apps/web/views/profile/RowTwo.tsx

```diff
@@ -0,0 +1,268 @@
+                  value={Number(mentorVal)}
+                  isDisabled={edit}
+                >
+                  <Stack spacing={5} direction="row">
+                    <Radio colorScheme="blue" value={1}>
+                      Yes
+                    </Radio>
+                    <Radio colorScheme="blue" value={0}>
+                      No
+                    </Radio>
+                  </Stack>
+                </RadioGroup>
+                <FormErrorMessage>{mentorError?.message}</FormErrorMessage>
+              </FormControl>
+            </Box>
+            {/* <Controller
+              control={control}
+              name="mentor"
+              render={({ field, fieldState: { error: mentorError } }) => (
+                <FormControl label="Mentor" isInvalid={!!mentorError} id="Mentor">
+                  <FormLabel>Can you be a Mentor</FormLabel>
+                  <RadioGroup {...field} isDisabled={edit}>
+                    <Stack spacing={5} direction="row">
+                      <Radio colorScheme="blue" value={1}>
+                        Yes
+                      </Radio>
+                      <Radio colorScheme="blue" value={0}>
+                        No
+                      </Radio>
+                    </Stack>
+                  </RadioGroup>
+                  <FormErrorMessage>{mentorError?.message}</FormErrorMessage>
+                </FormControl>
+              )}
+            /> */}
+          </Box>
+        )}
+        {role === 'Student' && (
+          <Box display="flex" flexDirection="column" justifyContent="space-between" mt="13px">
+            <Controller
+              control={control}
+              name="College"
+              render={({ field, fieldState: { error: collegeErr } }) => (
+                <FormControl label="College" isInvalid={!!collegeErr} id="College">
+                  <FormLabel>I currenlty study at</FormLabel>
+                  <AsyncSelect
+                    isDisabled={edit}
+                    {...field}
+                    isClearable
+                    loadOptions={getCollege}
+                    onInputChange={handleInputChange}
+                  />
+                  {collegeErr && <FormErrorMessage>Please pick an option</FormErrorMessage>}
+                </FormControl>
+              )}
+            />
+          </Box>
+        )}
+      </Box>
+      {role === 'Student' && (
+        <Box display="flex" flexDirection="column" justifyContent="space-between">
+          <Controller
+            control={control}
+            name="Passout"
+            render={({ field, fieldState: { error: descError } }) => (
+              <FormControl label="Passout" isInvalid={!!descError} id="Passout">
+                <FormLabel>Year of Passout</FormLabel>
+                <Select
+                  isDisabled={edit}
+                  options={[
+                    { label: dayjs().year(), value: dayjs().year() },
```

Comment:
Keep a variable at top for current year and use a map function to build an array with object label and value.